cclib/parser/fchkparser.py

```diff
@@ -171,6 +171,113 @@ def extract(self, inputfile, line):
         if line[0:11] == 'Shell types':
             self.parse_aonames(line, inputfile)
 
+        if line[0:19] == 'Real atomic weights':
+            count = int(line.split()[-1])
+            assert count == self.natom
+
+            atommasses = numpy.array(self._parse_block(inputfile, count, float, 'Atomic Masses'))
+
+            self.set_attribute('atommasses', atommasses)
+
+        if line[0:10] == 'SCF Energy':
+            self.scfenergy = float(line.split()[-1])
+
+            self.set_attribute('scfenergies', [utils.convertor(self.scfenergy,'hartree','eV')])
+
+        if line[0:18] == 'Cartesian Gradient':
+            count = int(line.split()[-1])
+            assert count == self.natom*3
+
+            gradient = numpy.array(self._parse_block(inputfile, count, float, 'Gradient'))
+
+            self.set_attribute('grads', gradient)
+
+        if line[0:25] == 'Cartesian Force Constants':
+            count = int(line.split()[-1])
+            assert count == (3*self.natom*(3*self.natom+1))/2
+
+            hessian = numpy.array(self._parse_block(inputfile, count, float, 'Gradient'))
+
+            self.set_attribute('hessian', hessian)
+
+        if line[0:13] == 'ETran scalars':
+            count = int(line.split()[-1])
+
+            etscalars = self._parse_block(inputfile, count, int, 'ET Scalars')
+
+            # Added new class attribute: net (number of excited estates)
+            self.net = etscalars[0]
+            self.netroot = etscalars[4]
+
+        if line[0:10] == 'ETran spin':
+            count = int(line.split()[-1])
+            assert count == self.net
+
+            etspin = self._parse_block(inputfile, count, int, 'ET Spin')
+
+            spin_labels = { 0:'Singlet',
+                            2:'Triplet',
+                           -1:'Unknown'}
+            etsyms = []
+            for i in etspin:
+                if i in spin_labels:
+                    etsyms.append(spin_labels[i])
+                else:
+                    etsyms.append(spin_labels[-1])
+
+            # This must be completed with actual symm irrep, but "ETran symm" section seems
+            # not to be given nothing but zeroes regardles the symm irrep
```

Comment:
This comment is confusing. As I understand it: "After this 'Etran sym' appears (and would need to be parsed), but at least in Gaussian this contains only zeroes regardless of the irrep."

Reply:
Ok, agree. I wanted to note that the extracted values will not coincide with those from Gaussian log files, as they lack the actual irrep label.